frontend/package.json

```diff
@@ -28,6 +28,7 @@
     "@turf/transform-scale": "^6.4.0",
     "@turf/truncate": "^6.4.0",
     "@webscopeio/react-textarea-autocomplete": "^4.7.3",
+    "RapiD": "facebookincubator/rapid#rapid-v1.1.8-tm.1",
```

Comment:
Will removing the previously installed RapiD dependency(rapid-v1.1.8_tm) be safe?

Reply:
It should be totally safe. Making the editors be different components instead of reloading the page required a few changes to RapiD as well.

Reply:
Removed on 27ab132bbea2d108fbe51c17f09b84622b9e78e0